.travis.yml

```diff
@@ -1,20 +0,0 @@
-language: ruby
-rvm:
-  - 3.0
-  - 2.7
-  - 2.6
-  - 2.5
-  - 2.4
-  - 2.3
-  - 2.2
-  - 2.1
-  - 2.0
-branches:
-  only:
-    - master
-matrix:
-  include:
-    # Run Danger only once, on 2.5
-    - rvm: 2.5
-      script: bundle exec danger
```

Comment:
Looks like `danger` got left behind as well. Can you include running this as its own job in the GitHub actions config?

Reply:
okay I'll add it